e2e-tests/IIoTPlatform-E2E-Tests/deploy/ConfigurationModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.deploy {
```

Comment:
We use camel case for directory names within E2E test project, please rename the directory from `deploy` to `Deploy`.

Reply:
+1

---
In reply to: [527059609](https://github.com/Azure/Industrial-IoT/pull/916#discussion_r527059609) [](ancestors = 527059609)

Reply:
fixed